bandcamp-downloader.py

```diff
@@ -193,6 +201,20 @@ def main() -> int:
         for link in links:
             download_album(link)
     CONFIG['TQDM'].close()
+
+    if args.unzip:
+        base_path = os.getcwd()
+        if args.directory:
+            base_path = args.directory
+        all_zip_files = glob.glob(f'{base_path}/**/*.zip', recursive=True)
```

Comment:
won't this unzip and delete any zip file it finds in `base_path` even if it wasn't added by `bandcamp-downloader.py`? 

what if you kept a list of downloaded files and only unzipped those rather than using the glob?

Reply:
yes you'd be right, any zip file in `base_path` would be unzipped which may not be the expected behaviour. 

I've pushed changes to unzip only the files downloaded by bandcamp-downloader, testing for both cases where ThreadPoolExecutor is and isn't used for parallel downloads.